src/pages/pool/_id.vue

```diff
@@ -192,6 +193,15 @@ watch(poolQuery.error, () => {
     removeAlert('pool-fetch-error');
   }
 });
+
+watch(
+  () => pool.value,
+  () => {
+    if (pool.value) {
+      metaService.setMeta(route, pool.value);
+    }
+  }
+);
```

Comment:
As far as I can tell, this doesn't work. Just run the preview URL for pool page through some OG metadata checkers. Of course it works to look at the page title when you're browsing, but that's about it. Maybe we need a different solution? For example, could we build a script that compiles a static json of poolId -> metadata that we can use in the routes? Will that even work because again it's being dynamically set after page load.

Reply:
actually for og this doesn't work, but it works for e.g googlebots and it is one of the tips here: 
https://developers.google.com/search/docs/crawling-indexing/javascript/javascript-seo-basics#titles-and-snippets